channel_config_test.go

```diff
@@ -0,0 +1,86 @@
+package stream_chat
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+)
+
+func TestDuration_MarshalJSON(t *testing.T) {
+	tests := []struct {
+		name  string
+		input PartitionTTL
```

Comment:
I'm guessing you meant to have `input` as a `DurationString` here?

Reply:
Whoops, good one. I renamed this type but didn't save the file and didn't notice before pushing

Fixed in 0dda39f556cc8910273d6d22fd58b54bdb15bfe7